src/js/components/Notification/__tests__/Notification-test.js

```diff
@@ -155,4 +159,78 @@ describe('Notification', () => {
     });
     expect(onClose).not.toHaveBeenCalled();
   });
+
+  test('custom theme', () => {
+    const onOpen = jest.fn();
+
+    const theme = {
+      notification: {
+        direction: 'row',
+        truncation: true,
+        container: {
+          pad: 'medium',
+        },
+        toast: {
+          direction: 'column',
+          truncation: false,
+        },
+        critical: {
+          background: 'red',
+          toast: {
+            background: 'background-front',
+          },
+        },
+      },
+    };
+
+    const Test = () => {
+      const [visible, setVisible] = useState(false);
+      return (
+        <Grommet theme={theme}>
+          <TestNotification status="critical" />
+          <Button
+            label="Show Toast Notification"
+            onClick={() => {
+              onOpen();
+              setVisible(true);
+            }}
+          />
+          {visible && (
+            <TestNotification toast title="Toast title" status="critical" />
+          )}
+        </Grommet>
+      );
+    };
+    const { asFragment } = render(<Test />);
+
+    expect(asFragment()).toMatchSnapshot();
+    userEvent.click(
+      screen.getByRole('button', { name: 'Show Toast Notification' }),
+    );
+    expect(screen.getByText('Toast title')).toBeInTheDocument();
+    expect(onOpen).toHaveBeenCalled();
+  });
+
+  test('actions', () => {
+    render(
+      <Grommet>
+        <TestNotification
+          actions={[{ href: '/some-link', label: 'Renew Subscription' }]}
+        />
+      </Grommet>,
+    );
+
+    const link = screen.getByRole('link', { name: 'Renew Subscription' });
+    expect(link).toHaveAttribute('href', '/some-link');
+  });
+
+  test('global', () => {
```

Comment:
Also need a test for `actions`. Just ensuring we have test coverage for all properties.

Reply:
actions is being tested in the test above.

Reply:
duh :)